mmdet/datasets/pipelines/transforms.py

```diff
@@ -75,13 +75,35 @@ def __init__(self,
 
     @staticmethod
     def random_select(img_scales):
+        """Randomly select an img_scale when ``multiscale_mode=='value'``
```

Comment:
Randomly select an image scale from the given candidates. (There is no restrict that it can only be used with ``multiscale_mode=='value'``)